components/brandprofile/ColorShowcase.tsx

```
@@ -0,0 +1,196 @@
+        )}
+      </button>
+
+      <div className="mt-2 space-y-1">
+        <p className="text-sm font-medium text-secondary-900 dark:text-secondary-100">
+          {color.name}
+        </p>
+        <button
+          onClick={() => handleCopy(color.hex)}
+          className="text-xs text-secondary-600 dark:text-secondary-400 hover:text-primary-600 dark:hover:text-primary-400 transition-colors cursor-pointer"
+          tabIndex={0}
+        >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)